sdk/include/reactos/shellutils.h

```diff
@@ -98,7 +98,7 @@ static inline UINT
 SHELL_ErrorBoxHelper(HWND hwndOwner, UINT Error)
 {
     WCHAR buf[400];
-    UINT cch, u32_errstr = 2;
+    UINT cch, user32_IDS_ERROR = 2;
```

Comment:
Still a magic constant.

Reply:
How would it be less magic if it was a define?

Reply:
You can use a newly-added parameter.

Reply:
> You can use a newly-added parameter

I have no idea what this even means in this context

Reply:
If this needs to be reused elsewhere, adding a `#define IDS_ERROR 2` in our `sdk/include/reactos/undocuser.h` would make sense (+ a comment about what this define is for).

Reply:
> If this needs to be reused elsewhere

That is a big if. 

Kernel32 and advapi32 should not be displaying a lot of message boxes. Other components higher up?

Cross that bridge when we get there?

Reply:
Agreed :+1: